src/app/item/add/page.tsx

```diff
@@ -46,75 +83,79 @@ const AddItemForm = () => {
-              margin="normal"
-              value={lowValue}
-              onChange={(e) =>
-                setLowValue(Number(e.target.value) < 0 ? '0' : e.target.value)
-              }
-            />
-          </Grid>
-          <Grid item xs={12}>
-            <Button
-              type="submit"
-              variant="contained"
-              color="primary"
-              fullWidth
-              sx={{
-                height: 56,
-                textTransform: 'none',
-                fontSize: 'large',
-                backgroundColor: 'rgba(55, 149, 65, 0.8)',
-              }}
-            >
-              Add Donation Item
-            </Button>
+        <form onSubmit={handleSubmit}>
+          <Grid container spacing={3} justifyContent="flex-start">
+            <Grid item xs={12}>
+              <TextField
+                required
+                label="Item Name"
+                variant="outlined"
+                fullWidth
+                margin="normal"
+                value={itemName}
+                onChange={(e) => setItemName(e.target.value)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                required
+                label="Category"
+                variant="outlined"
+                fullWidth
+                margin="normal"
+                value={category}
+                onChange={(e) => setCategory(e.target.value)}
+              />
+              {/* Consider using Select component here for dropdown */}
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                type="number"
+                id="highValue"
+                label="High Value"
+                variant="outlined"
+                fullWidth
+                margin="normal"
+                value={highValue}
+                onChange={(e) =>
+                  setHighValue(
+                    Number(e.target.value) < 0 ? '0' : e.target.value
+                  )
+                }
+              />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                type="number"
+                id="lowValue"
+                label="Low Value"
+                variant="outlined"
+                fullWidth
+                margin="normal"
+                value={lowValue}
+                onChange={(e) =>
+                  setLowValue(Number(e.target.value) < 0 ? '0' : e.target.value)
+                }
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <Button
+                type="submit"
+                variant="contained"
+                color="primary"
+                fullWidth
+                sx={{
+                  height: 56,
+                  textTransform: 'none',
+                  fontSize: 'large',
+                  backgroundColor: 'rgba(55, 149, 65, 0.8)',
+                }}
+              >
+                Add Donation Item
```

Comment:
This should be replaced with "Add Item" or "Add Pricing Guide". For now just set it to "Add Item"

Reply:
Made requested changes